veloroutes_voies_vertes/install/sql/export_database_structure_to_SQL.sh

```diff
@@ -59,8 +59,8 @@ done
 rm "$OUTDIR/dump"
 
 # NOMENCLATURE
-#echo "GLOSSARY"
-#if [ $SCHEMA = 'adresse' ]
-#then
-#    pg_dump service=gobs --data-only --inserts --column-inserts -n $SCHEMA --no-acl --no-owner --table "gobs.glossary" -f "$OUTDIR"/90_GLOSSARY.sql
-#fi
+echo "GLOSSARY"
+if [ $SCHEMA = 'veloroutes' ]
+then
+   pg_dump service=vvv --data-only --inserts --column-inserts -n $SCHEMA --no-acl --no-owner --table "veloroutes.*_val" -f "$OUTDIR"/90_GLOSSARY.sql
```

Comment:
```suggestion
   pg_dump service=$SERVICE --data-only --inserts --column-inserts -n $SCHEMA --no-acl --no-owner --table "veloroutes.*_val" -f "$OUTDIR"/90_GLOSSARY.sql
```
Il faut utiliser la variable du script

Reply:
Tu peux aussi supprimer l'utilisation du `rename` et faire un `mv`

Reply:
plus de soucis avec le `rename` @Enola-sengeissen

Reply:
j'ai testé j'ai toujours ces erreurs:

`sed: 1: "veloroutes/70_COMMENT.sql": invalid command code v
mv: rename s#\|#_#g to veloroutes/70_COMMENT.sql: No such file or directory`

la commande sed pose également problème, mv remplace sed & rename ou que rename?

Reply:
problem solved pour sed, apparemment sous macOS c'est mieux d'écrire :
`sed **-i "" -e** "s#SET idle_in_transaction_session_timeout = 0;##g" "$OUTDIR"/"$I"_"$ITEM".sql;`

Reply:
problem solved pour sed, apparemment sous macOS c'est mieux d'écrire :

`sed` -i "" -e "s#SET idle_in_transaction_session_timeout = 0;##g" "$OUTDIR"/"$I"_"$ITEM".sql;`